remixer_example/src/main/java/com/google/android/apps/remixer/RemixerApplication.java

```diff
@@ -31,10 +32,21 @@
  */
 public class RemixerApplication extends Application {
 
+  // Keep this off so there is no need for proper Firebase credentials for the example app out of
+  // the box. You can change it to true if you update the google-services.json file.
+  private static final boolean USE_FIREBASE_REMOTE_CONTROLLER = false;
+
   @Override
   public void onCreate() {
     super.onCreate();
     RemixerInitialization.initRemixer(this);
-    Remixer.getInstance().setSynchronizationMechanism(new LocalStorage(getApplicationContext()));
+    if (USE_FIREBASE_REMOTE_CONTROLLER) {
+      FirebaseRemoteControllerSyncer syncer =
+          new FirebaseRemoteControllerSyncer(getApplicationContext());
```

Comment:
is there a requirement that the context you pass in be the application context?

If so, I would recommend you pass in `this` here, and in new FirebaseRemoteControllerSyncer(), set `context = context.getApplicationContext()`.

That way, you're protected against any non-app contexts.